### PR TITLE
replaces corporate climber theft objective with a new custom objective

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

https://user-images.githubusercontent.com/95004236/190528842-3a9bb337-5326-4c57-b249-d3a2542936a4.mp4
ur objective is to connect the uplink to your employer's HQ within the area specified. wether u choose to buy some of the items from the uplink is up 2 u

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: fulp_modules/features/antagonists/infiltrators/infil_corpmarket.dm [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+	name = "Security Auto Rifle"
+	cost = 30
+	item = /obj/item/gun/ballistic/automatic/wt550
+	desc = "A model discontinued by Nanotrasen, we managed to find a whole crate of it just floating in space!"
+
+/datum/infil_corpitem/ammo/sniper_ammo
+	name = "Sniper Rounds"
+	cost = 10
+	item = /obj/item/ammo_box/magazine/sniper_rounds
+	desc = "An extra round for the Sniper rifle."
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/infil_corpmarket.dm:74]: one extra round
actually is an entire magazine????

File: fulp_modules/features/antagonists/infiltrators/infil_items.dm [hunk 2]
```diff
@@ -445,3 +445,47 @@
 
 /mob/living/silicon/robot/model/infiltrator/ResetModel()
 	self_destruct(src)
+
+
+/obj/item/antag_spawner/nuke_ops/infiltrator_backup
+	name = "Corporate Yes-man beacon"
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/infil_items.dm:451]:
```suggestion
	name = "\improper Corporate Yes-man beacon"
```

File: fulp_modules/features/antagonists/infiltrators/infil_items.dm [hunk 3]
```diff
@@ -445,3 +445,47 @@
...
+	to_chat(user, span_notice("You activate [src] and wait for confirmation."))
+	var/list/infil_candidates = poll_ghost_candidates("Do you want to play as an infiltrator backup?", ROLE_INFILTRATOR, ROLE_INFILTRATOR, 150, POLL_IGNORE_SYNDICATE)
+	if(LAZYLEN(infil_candidates))
+		used = TRUE
+		var/mob/dead/observer/ghost = pick(infil_candidates)
+		spawn_antag(ghost.client, get_turf(src), "infiltrator", user.mind)
+		do_sparks(4, TRUE, src)
+		qdel(src)
+	else
+		to_chat(user, span_warning("Unable to contact corporate. Please wait and try again later."))
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/infil_items.dm:472]:
```suggestion
		to_chat(user, span_warning("Unable to contact Corporate. Please wait and try again later."))
```

**Comment by SgtHunk** [comment]
The author of this pull request has unfortunately passed away, this'll be merged to honor him.

**Pull request merged**